<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWarningDefine</class>
 <widget class="QDialog" name="DialogWarningDefine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预警设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DialogWarningDefine{
border-image: url(:/widget/调距桨背景.png);
}

QGroupBox{
	background-color: rgba(16, 57, 70, 1);
	border: 0px solid rgb(111, 255, 255);
	margin-top:3.3ex;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QGroupBox::title{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	background-color: rgba(19, 67, 79, 1);
	padding: 7 1000px;
}

QLineEdit{
	background-color: rgba(255, 255, 255,0.2);
	border-width:0;
	border-style:outset;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QLabel{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton::indicator{
	border-color: rgba(255, 255, 255,0);
	background:transparent;
}


QRadioButton::indicator:checked{
	border-image:url(:/widget/images/green.png)
}

QRadioButton::indicator:unchecked{
	border-image:url(:/widget/images/unchecked.png)
}

QPushButton{
	border: 2px solid rgba(61, 255, 255,0.4);
	background-color: rgba(12, 34, 45, 0.6);
	font: 12pt &quot;微软雅黑&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
}

QPushButton::disabled{
	border: 0px;
	color: rgba(102, 249, 247, 0.3);
	text-align：center;
}

QPushButton::pressed{
	border: 2px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QPushButton::hover{
	border: 2.5px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 0.6);
	font: 75 12pt &quot;微软雅黑&quot;;
}

QDoubleSpinBox{
	border: 0px solid rgba(61, 255, 255,0.4);
	background-color: rgba(255, 255, 255, 0.2);
	font: 12pt &quot;Times New Roman&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
	padding-left:10px;
}


QTabWidget{
	background-color: rgba(19, 67, 79, 1);
	border-color: rgba(19, 67, 79, 1);
	border:0px;
	margin:0px;
}

QTabWidget::pane{
	border:0px;
	margin:0px;
}

QTabWidget::tab-bar {
	left: 2px;
	border:0px;
	margin:0px;
}

QTabBar::tab:first {
	background-color: rgba(11, 49, 61, 1);
	border-color: rgba(11, 49, 61, 1);
	color: rgba(102, 249, 247, 1);
	min-width: 4em;
	min-height:1em;
}
QTabBar::tab {
	font: 12pt &quot;微软雅黑&quot;;
	background-color: rgba(11, 49, 61, 255);
	border-color: rgba(11, 49, 61, 255);
	color: rgba(102, 249, 247, 1);
	min-width: 4em;
	min-height:1em;
}

QTabBar::tab:first:selected {
	min-width: 4em;
	min-height:1em;
}

QTabBar::tab:selected {
margin-bottom:0.2em;
}

QTabBar::tab:!selected{
	min-width: 4em;
	min-height:1em;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_30">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>工况名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_RuleName">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	background-color: rgba(255, 255, 255,1);
	border-width:0;
	border-style:outset;
	color: rgb(0, 0, 0);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_31">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>工况描述：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_RuleDescription">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	background-color: rgba(255, 255, 255,1);
	border-width:0;
	border-style:outset;
	color: rgb(0, 0, 0);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="label_38">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>编辑人：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Editor">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	background-color: rgb(150, 150, 150);
	border-width:0;
	border-style:outset;
	color: rgb(0, 0, 0);
	font: 12pt &quot;微软雅黑&quot;;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>工况特征变量</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget_TaskFeaturePara">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0.2);</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="Btn_TaskFeatureMod">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_TaskFeatureDelete">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>工况关联变量</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget_TaskAssoPara">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255,0.2);</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="Btn_TaskAssoMod">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_TaskAssoDelete">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_OK">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Cancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
